pkg/objectstate/loglevel/loglevel.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
this is looks like a fine package, but I don't think it belongs to `objectstate`. Let's move up one level:
`pkg/loglevel/...`